README.md

```diff
@@ -107,171 +114,173 @@ Server running... press ctrl-c to stop.
+      - /community/
+      - /swagger/
+      - /vagrant/
+      - /guides/m1x/
+      - /guides/v2.0/
+      - /guides/v2.1/
+    # - /guides/v2.2/
+      - /guides/v2.3/
 
-To stop the server, press `Ctrl+C` in the serve terminal.
+    # Excluded in config.yml
+      - /scss/
+      - /bin/
+      - /node_modules/
+      - /vendor/
+      - /.*
+      - /Rakefile
+    ```
 
-> ***TIP***
-> Leave the serve terminal open and running. Every time you save changes to a file, it automatically regenerates the site so you can test the output immediately. Using the `--incremental` option limits re-generation to pages that have changed. `--livereload` refreshes the changed HTML page in your browser after each cycle of re-generation.
+1.  Preview the site using the `rake preview` command.
 
-### How to minimize build time locally
+If you do not have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake tool generates all versions of the documentation.
 
-Generation of the entire website can take up to 15 minutes depending on your environment.
-If you want to preview only selected directories, you can exclude all directories that you don't need using additional configuration file `_config.local.yml`, which is tracked by `rake preview` task by default.
+To disable link checking, add `check_links: false` to the `_config.local.yml` file.
 
-1. Create a `_config.local.yml` file at the root of the project directory and exclude all versions except the one that you want to preview.
-   The following example will generate Magento 2.2 documentation and the documentation located at the root of the project such as `mftf`.
+To ignore the `_config.local.yml` file, use the `rake preview:all` command.
 
-   ```yaml
-    exclude:
-     - /community/
-     - /swagger/
-     - /vagrant/
-     - /guides/m1x/
-     - /guides/v2.0/
-     - /guides/v2.1/
-    # - /guides/v2.2/
-     - /guides/v2.3/
+## Build using Vagrant
 
-    # Excluded in config.yml
-     - /scss/
-     - /bin/
-     - /node_modules/
-     - /vendor/
-     - /.*
-     - /Rakefile
-   ```
+Optionally, you can build the DevDocs site using [Vagrant](https://github.com/magento-devdocs/vagrant-for-magento-devdocs).
+
+## Build DevDocs in Windows
 
-2. Run [`rake preview`](#using-rake)
+Some of the technologies we use to develop DevDocs are not compatible with Windows, such as [Jekyll](https://jekyllrb.com/docs/windows/). For this reason, we do not support DevDocs management in Windows; however, we have documented the following procedures to build the DevDocs in a Windows environment. Any further use of this setup or troubleshooting is up to you. 
 
-If you don't have the `_config.local.yml` file at the root of your `devdocs/` directory, the rake will generate all versions of the documentation.
+Download software:
 
-To disable link checking, add `check_links: false` to `_config.local.yml`.
+-  [Git for Windows](https://gitforwindows.org)
+-  [Chocolatey](https://chocolatey.org/install)
```

Comment:
This link contains all the guidelines listed in the next section.
When we quote the guidelines we automatically take responsibility for maintaining them.